addons/test_mail/tests/test_performance.py

```diff
@@ -472,7 +472,7 @@ def test_mail_composer_w_template(self):
         test_record, test_template = self._create_test_records()
         test_template.write({'attachment_ids': [(5, 0)]})
 
-        with self.assertQueryCount(admin=27, employee=27):  # tm 16/16 / com 26/26
+        with self.assertQueryCount(admin=28, employee=28):  # tm 16/16 / com 26/26
```

Comment:
so in order to validate partner's existence on the peppol network on the fly when they update partners' details, we have to call `button_account_peppol_check_partner_endpoint`, where we need to check the `edi_mode` to determine what to query:
- demo -> always valid by default
- test -> query test peppol network
- prod -> query prod peppol network

that information is only accessible by accessing `account_edi_proxy_client_ids` on the company, which adds one more query.

an alternative would be to just reset validity whenever eas/endpoint/ubl format changes, but in my opinion that is not the most user-friendly solution.